packages/aws-cdk-lib/aws-cloudwatch/test/dashboard.test.ts

```diff
@@ -230,6 +238,80 @@ describe('Dashboard', () => {
     const template = Annotations.fromStack(stack);
     template.hasWarning('/MyStack/MyDashboard', Match.stringLikeRegexp("Math expression 'oops' references unknown identifiers"));
   });
+
+  test('has different variables', () => {
```

Comment:
lets split this unit test up into different tests that all test one specific use case.

Reply:
The tests are now all split